cfme/scripting/sprout.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""Script to encrypt config files.
+
+Usage:
+
+   scripts/encrypt_conf.py confname1 confname2 ... confnameN
+   scripts/encrypt_conf.py credentials
+"""
+import click
+import signal
+import time
+import sys
+# import yaml
+
+from cfme.test_framework.sprout.plugin import SproutManager, SproutProvisioningRequest
+# from utils.path import conf_path
+
+
+@click.group(help='Functions for interacting with sprout')
+def main():
+    pass
+
+
+@main.command('checkout', help='Checkout appliance and start keepalive daemon')
+@click.option('--appliances', default=1, help='How many appliances to provision')
+@click.option('--timeout', default=60, help='How many minutes is the lease timeout')
+@click.option('--provision-timeout', default=60,
+              help='How many minutes to wait for appliances provisioned')
+@click.option('--group', default=None, help='Which stream to use')
+@click.option('--version', default=None, help='Which version to use')
+@click.option('--date', default=None, help='Which date to use')
+@click.option('--desc', default=None, help='Set description of the pool')
```

Comment:
How cheeky would it be to default this to `id -un` and some alphanumeric or date string.

Reply:
Later